citrination_client/client.py

```diff
@@ -191,43 +191,43 @@ def predict(self, model_name, candidates):
     def _get_predict_url(self, model_name):
         return self.api_url + '/csv_to_models/' + model_name + '/predict'
 
-    def upload_file(self, file_path, data_set_id, root_path=None):
+    def upload(self, dataset_id, source_path, dest_path=None):
         """
-        Upload file to Citrination.
-
-        :param file_path: File path to upload.
-        :param data_set_id: The dataset id to upload the file to.
-        :return: Response object or return code if the file was not uploaded.
+        Upload a file, specifying source and dest paths a file (acts as the scp command)
+        :param source_path: The path to the file on the source host
+        :param dest_path: The path to the file where the contents of the upload will be written (on the dest host)
+        :return: The response object or a boolean False if the file was not uploaded
```

Comment:
Is it true that a boolean is returned?  It looks like a JSON message is returned in all cases.
(Out of interest, why do we return JSON?  All values are memory-local and not going over the wire, aren't they?)